src/runtime/execution.rs

```diff
@@ -429,6 +434,16 @@ impl ExecutionState {
     }
 }
 
+/// The number of context switches that happened so far in the current Shuttle execution.
+///
+/// Note that this is the number of *possible* context switches, i.e., including times when the
+/// scheduler decided to continue with the same task.
+///
+/// Panics if called outside of a Shuttle execution.
+pub fn context_switches() -> usize {
```

Comment:
Not super important, but I don’t really like bare functions like this. I’d consider implementing this as a method on `ExecutionState`, in the same vein as `me()`, and then put the public implementation in lib.rs that just calls that method.

Reply:
Will do.